fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanApplicationWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -365,11 +387,106 @@ public CommandProcessingResult submitApplication(final JsonCommand command) {
                 createAndPersistCalendarInstanceForInterestRecalculation(newLoanApplication);
             }
 
+
+            // loan account number generation
+            String accountNumber="";
```

Comment:
Can we align GLIM creation with that of GSIM creation, I dont' see bulk creation in GLIM.